assets/sass/okta-theme.scss

```diff
@@ -57,7 +57,7 @@
 
 
 // scss-lint:disable IdSelector
-#okta-sign-in {
+#okta-sign-in.auth-container {
```

Comment:
I think all will still work without adding ```.auth-container``` since they are the same element, but in any case, it doesn't hurt.